reactive-banana/src/Reactive/Banana/Prim/Mid/Plumbing.hs

```diff
@@ -187,17 +187,17 @@ keepAlive child parent = liftIO $ void $
 
 addChild :: SomeNode -> SomeNode -> Build ()
 addChild parent child =
-    RW.tell $ BuildW ([InsertEdge parent child], mempty, mempty, mempty)
+    RW.tell mempty { bwDependencyChanges = [InsertEdge parent child] }
 
 changeParent :: Pulse child -> Pulse parent -> Build ()
 changeParent pulse0 parent0 =
-    RW.tell $ BuildW ([ChangeParentTo pulse parent], mempty, mempty, mempty)
+    RW.tell mempty { bwDependencyChanges = [ChangeParentTo pulse parent] }
    where
      pulse = _nodeP pulse0
      parent = _nodeP parent0
 
 liftIOLater :: IO () -> Build ()
-liftIOLater x = RW.tell $ BuildW (mempty, mempty, Action x, mempty)
+liftIOLater x = RW.tell mempty { bwLateIO = Action x }
```

Comment:
```suggestion
liftIOLater x = RW.tell emptyBuildW{ bwLateIO = Action x }
```

Two requests on the matter of record syntax:

* `mempty` is too generic a name. Could you define `emptyBuildW = mempty`, so that it better resembles the `BuildW` constructor?
* Record syntax messes a little with the "spaces separate distinct arguments" rule. Could you remove the spaces between the value and `{`? In this way, it becomes visually clear that `{` modifies the value.

Reply:
You got it!